### PR TITLE
explore: format x-axis labels and tick positions

### PR DESCRIPTION
**Pull request by pnavarrc**
Small fix to adjust tick positions to what we have in [the mocks](https://www.figma.com/file/nkoC6P02eXxkB95ukdU6Ms/Explore?node-id=260%3A24)

image

### CODE REVIEW

File: src/components/Explore/ExploreChart.tsx [hunk 1]
```diff
@@ -79,17 +80,20 @@ const ExploreChart: FunctionComponent<{
...
       <Group key="axes" top={marginTop} left={marginLeft}>
         <Styles.GridLines>
-          <Grid
-            xScale={timeScale}
-            yScale={yScale}
-            width={innerWidth}
-            height={innerHeight}
-          />
+          <GridColumns<Date> scale={timeScale} height={innerHeight} />
+          <GridRows<number> scale={yScale} width={innerWidth} />
```
Comment (pnavarrc) [inline src/components/Explore/ExploreChart.tsx:84]: `@vx/grid` has a bug that doesn't allow to pass specific values to the scale. It's picking the correct values somehow (I _think_ that the axis component mutates the scale ticks, and then the grid takes it from the scale). I'm going to dig into this.

Comment (pnavarrc) [reply src/components/Explore/ExploreChart.tsx:84]: (later)

**Comment by mikelehen** [comment]
Though I'm happy to defer to Josh, you or others if you think it's fine. 😁

**Comment by pnavarrc** [comment]
The mobile version will have specific controls to manage that, however, it wouldn't look good in the "All" setting. I will update the format to make it mobile friendly and update the PR 👍 

< img src="https://user-images.githubusercontent.com/114084/90179545-72c0e880-dd62-11ea-8bc2-5bc075d81a20.png" width="375" />

**Comment by pnavarrc** [comment]
Updated!

**Pull request merged**